website/docs/networking/vpc-lattice/latticeabtesting.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "A/B testing with VPC Lattice"
+sidebar_position: 20
+---
+
+In this section we will show how to use Amazon VPC Lattice to perform A/B testing gradually shifting traffic from the `checkout` service to a new version `checkout-v2`.
+
+# Set up Lattice Service Network
+
+1. Create the Kubernetes Gateway `eks-workshop-gw`:
+   ```bash
+   $ kubectl apply -f workspace/modules/networking/vpc-lattice/controller/eks-workshop-gw.yaml
+   ```
+1. Verify that `eks-workshop-gw` gateway is created:
+   ```bash
+   $ kubectl get gateway  
+   ```
+   ```
+   $ NAME              CLASS         ADDRESS   READY   AGE
+   $ eks-workshop-gw   aws-lattice                     12min
+   ```
+
+1. Once the gateway is created, find the VPC Lattice service network. Wait until the status is `Reconciled` (this could take about five minutes).
+   ```bash
+   $ kubectl describe gateway eks-workshop-gw
+   ```
+   ```
+   $ apiVersion: gateway.networking.k8s.io/v1alpha2
+   kind: Gateway
+   ...
+   $ status:
+   conditions:
+   message: 'aws-gateway-arn: arn:aws:vpc-lattice:us-west-2:<YOUR_ACCOUNT>:servicenetwork/sn-03015ffef38fdc005'
+   reason: Reconciled
+   status: "True"
+   ```
+ Now you can see the associated Service Network created in the VPC console under the Lattice resources.
+![Checkout Service Network](assets/servicenetwork.png)
+
+# Create Routes to targets
+We will show how to preform A/B testing between the two version (`checkout`and `checkoutv2`) using `HTTPRoutes`.
+
+**NOTE:** (At the time of writing, the `targetPort` has to be updated  to `8080` for the Lattice Controller to reroute the traffic). This will be improved in future releases.
```

Comment:
Leaves the reader really hanging ... what limitation is this? What was the previous value? Other details to educate the reader ...

Reply:
Hi Umair, for the moment this is a bug of the controller: https://github.com/aws/aws-application-networking-k8s/issues/86.

Reply:
Sorry Federica, my intent was not clear here. My suggestion is to give the reader an idea what this comment means with a little explanation and linking to the issue. Without it, the reader can be puzzled by what's going on.

Reply:
changed